src/exponential_utils.jl

```diff
@@ -468,7 +468,8 @@ Non-allocating version of `phiv_timestep`.
 """
 function phiv_timestep!(u::Vector{T}, t::Float64, A, B::Matrix{T}; tau::Float64=0.0, 
```

Comment:
`t` isn't necessarily Float64

Reply:
By this do you mean it can be other `Real` type or in general complex? I'm not sure if everything will work out OK for complex time though.

Reply:
any real